.ado/variables/vars.yml

```
@@ -1,6 +1,6 @@
 variables:
   VmImageApple: macOS-13
-  slice_name: 'Xcode_15.0'
-  xcode_version: '/Applications/Xcode_15.0.app'
+  slice_name: 'Xcode_15.2'
+  xcode_version: '/Applications/Xcode_15.2.app'
   ios_version: '17.0.1'
```

This should also be bumped:
```suggestion
  ios_version: '17.2'
```